src/test/java/org/fisco/bcos/web3j/abi/TypeEncoderTest.java

```diff
@@ -257,4 +259,79 @@ public void testEmptyArray() {
                 TypeEncoder.encodeDynamicArray(array),
                 is("0000000000000000000000000000000000000000000000000000000000000000"));
     }
+
+    @Test
+    public void throwsIfMaxOfU256() {
+        try {
+            Uint value = new Uint256(BigInteger.valueOf(-1));
+            TypeEncoder.encodeNumeric(value);
+        } catch (UnsupportedOperationException e) {
+            MatcherAssert.assertThat(
+                    e.getMessage(),
+                    equalTo(
+                            "Bitsize must be 8 bit aligned, and in range 0 < bitSize <= 256, and in valid range."));
+        }
+
+        try {
+            Uint value =
+                    new Uint256(
+                            new BigInteger(
+                                    "7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                                    16));
+            TypeEncoder.encodeNumeric(value);
+        } catch (UnsupportedOperationException e) {
+            MatcherAssert.assertThat(
+                    e.getMessage(),
+                    equalTo(
+                            "Bitsize must be 8 bit aligned, and in range 0 < bitSize <= 256, and in valid range."));
+        }
+    }
+
+    @Test
+    public void throwsIfMinOfInt256() {
+        try {
+            Int value =
+                    new Int(
+                            new BigInteger(
+                                    "-ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                                    16));
+            TypeEncoder.encodeNumeric(value);
+        } catch (UnsupportedOperationException e) {
+            MatcherAssert.assertThat(
+                    e.getMessage(),
+                    equalTo(
+                            "Bitsize must be 8 bit aligned, and in range 0 < bitSize <= 256, and in valid range."));
+        }
+
+        try {
+            Uint value =
+                    new Uint256(
+                            new BigInteger(
+                                    "-77fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+                                    16));
+            TypeEncoder.encodeNumeric(value);
+        } catch (UnsupportedOperationException e) {
+            MatcherAssert.assertThat(
+                    e.getMessage(),
+                    equalTo(
+                            "Bitsize must be 8 bit aligned, and in range 0 < bitSize <= 256, and in valid range."));
+        }
+    }
+
+    @Test
+    public void throwsIfMaxOfInt256() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732356)